bccsp/pkcs11/pkcs11.go

```diff
@@ -84,25 +84,42 @@ func loadLib(lib, pin, label string) (*pkcs11.Ctx, uint, *pkcs11.SessionHandle,
 func (csp *impl) getSession() (session pkcs11.SessionHandle) {
 	select {
 	case session = <-csp.sessions:
-		logger.Debugf("Reusing existing pkcs11 session %+v on slot %d\n", session, csp.slot)
```

Comment:
You've removed the Debug statement on the happy path. Would be nice to keep so you can follow the flow of execution

Reply:
Added the debug statement back